<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paints</class>
 <widget class="QDialog" name="Paints">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Укажите параметры краски</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeCmbx">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Для потолка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Латексная краска</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Масляная краска</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Резиновая краска</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Полиуретановая краска</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Огнезащитная краска</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Алкидная краска</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Водоэмульсионная краска</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Силикатная краска</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>По штукатурке</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>По шпаклевке</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>По короеду</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>По ржавчине</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>По дереву</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>По бетону</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>По металлу</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnCalc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество краски (г):</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputPaintsWeight">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название краски</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
